docs/bin/default.dot

```diff
@@ -37,6 +37,7 @@ N16;
 N17;
 N18;
 N19;
```

Comment:
FYI changes in the docs folder are automatically generated with `yarn prepare` in the docs folder